src/frontend/screens/Settings/sections/GamesSettings/index.tsx

```diff
@@ -90,15 +94,15 @@ export default function GamesSettings() {
 
         <AlternativeExe />
 
-        <ShowFPS />
+        {usesWine && <ShowFPS />}
```

Comment:
This one needs to appear on macOS as well. Would be good to check how settings are there before merging it.

Reply:
oops! I fixed that and a few more, I also moved that logic inside the component so the component is also controls if it's shown (the logic was there but it depended in some location state that is not that reliable, not it uses the current game's game info)